Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM node:14.17.1-alpine3.11 AS build
+FROM node:14.17-alpine3.14 AS build
 
 # Set working directory
 WORKDIR /usr/src/app
 
 # Install dependencies
 RUN apk add --no-cache \
-    python=~2.7.18 \
-    make=~4.2.1
+    python2=~2.7 \
```

Comment:
Why python2? Its EOL since start 2020 see https://www.python.org/doc/sunset-python-2/

Reply:
I found my old (pre-GitHub) PR that introduced these, with the note:

> - Add dependencies Python & Make, as some packages needed them to be installed

...not that explanatory. Shame on me.

I tried removing these and it built successfully. These dependencies does not seem relevant anymore.